Content.Shared/AirlockPainter/Components/PaintableAirlockComponent.cs

```diff
@@ -0,0 +1,9 @@
+namespace Content.Server.AirlockPainter
```

Comment:
```suggestion
using Content.Shared.AirlockPainter.Prototypes;
using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;

namespace Content.Server.AirlockPainter
```

Sorry should have also mentioned in the custom type serializer that it also needs name-spaces and the suggestion couldn't just be applied directly.